<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <arg name="selector_component_input" default="/planning/candidate/trajectories"/>
  <arg name="trajectory_concatenator_param_path" default="$(find-pkg-share autoware_trajectory_concatenator)/config/trajectory_concatenator.param.yaml"/>

  <node pkg="autoware_trajectory_concatenator" exec="autoware_trajectory_concatenator_node" name="trajectory_concatenator_node" output="screen">
      <param from="$(var trajectory_concatenator_param_path)" allow_substs="true"/>
      <remap from="~/input/trajectories" to="$(var selector_component_input)"/>
      <remap from="~/input/odometry" to="/localization/kinematic_state"/>
      <remap from="~/input/selected_trajectory" to="$(var control_component_input)"/>
      <remap from="~/output/trajectories" to="/planning/trajectory_selector/concatenate/trajectories"/>
  </node>
</launch>
